modules/37_industry/subsectors/datainput.gms

```diff
@@ -317,32 +317,47 @@ pm_ue_eff_target("ue_otherInd")         = 0.008;
 
 
 
-*** FS: CES markup cost industry
-*** default values of CES markup
-p37_CESMkup(t,regi,in) = 0;
+*** CES mark-up cost industry
```

Comment:
also this comments might go into the automated documentation. If they are the same for all realizations they should go to the module description

Reply:
I agree. The module and realization descriptions are quite short and would need some extension anyways. Let's tackle this in a future cleaning session. I created this [issue](https://github.com/remindmodel/development_issues/issues/157).